uber/models.py

```diff
@@ -1690,7 +1690,7 @@ def merch(self):
             if self.paid_for_a_swag_shirt:
                 shirt = 'a 2nd ' + shirt
             if not self.volunteer_swag_shirt_earned:
-                shirt += ' (tell them they will be reported if they take their shirt then do not work their shifts)'
+                shirt += ' (this volunteer must work at least 6 hours or they will be reported for picking up their shirt)'
```

Comment:
This message originally only displayed while handing out merch, but since we're now adding it to the details page, I changed the wording a bit.